README.md

```diff
@@ -204,10 +208,16 @@ ambient occlusion is disabled.
 sky texture that surrounds the scene. The texture is not used for
 baking.
 
+`ambientLight` has the following properties:
+
++ `strength` optional, a float that specifies how strong is the effect
+of ambient light, defaults to `0.05`. `0` is an equivalent of
```

Comment:
a float that specifies ambient light strength

Reply:
OK!